tests/drum/test_fit.py

```diff
@@ -122,10 +122,32 @@ def test_fit_for_use_output_and_nested(
         )
 
     @pytest.mark.parametrize(
-        "framework", [RDS, SKLEARN_BINARY, SKLEARN_REGRESSION, XGB, KERAS, PYTORCH]
+        "framework, problem, docker",
+        [
+            (RDS, BINARY_TEXT, None),
+            (RDS, REGRESSION, None),
+            (RDS, ANOMALY, None),
+            (RDS, MULTICLASS, None),
+            (SKLEARN_BINARY, BINARY_TEXT, DOCKER_PYTHON_SKLEARN),
+            (SKLEARN_REGRESSION, REGRESSION, DOCKER_PYTHON_SKLEARN),
+            (SKLEARN_BINARY, BINARY_TEXT, None),
+            (SKLEARN_REGRESSION, REGRESSION, None),
+            (SKLEARN_ANOMALY, ANOMALY, None),
+            (SKLEARN_MULTICLASS, MULTICLASS, None),
+            (XGB, BINARY_TEXT, None),
+            (XGB, REGRESSION, None),
+            (XGB, ANOMALY, None),
+            (XGB, MULTICLASS, None),
+            (KERAS, BINARY_TEXT, None),
+            (KERAS, REGRESSION, None),
+            (KERAS, ANOMALY, None),
+            (KERAS, MULTICLASS, None),
+            (PYTORCH, BINARY_TEXT, None),
+            (PYTORCH, REGRESSION, None),
+            (PYTORCH, ANOMALY, None),
+            (PYTORCH, MULTICLASS, None),
+        ],
     )
-    @pytest.mark.parametrize("problem", [BINARY_TEXT, REGRESSION, ANOMALY])
-    @pytest.mark.parametrize("docker", [DOCKER_PYTHON_SKLEARN, None])
     @pytest.mark.parametrize("weights", [WEIGHTS_CSV, WEIGHTS_ARGS, None])
```

Comment:
since weights_csv and weights_args pretty much do the same thing for hte estimator, we don't really need both. This can be thought thru further though